library/unwind/build.rs

```diff
@@ -4,7 +4,9 @@ fn main() {
     println!("cargo:rerun-if-changed=build.rs");
     let target = env::var("TARGET").expect("TARGET was not set");
 
-    if cfg!(feature = "llvm-libunwind")
+    if cfg!(feature = "system-llvm-libunwind") {
+        println!("cargo:rustc-link-lib=unwind");
```

Comment:
This won't work as you expect since `gcc_s` is linked in library/unwind/lib.rs and should be changed there.

Reply:
Huh? Seems to work in my tests though?

I can try to move it in lib,rs though

Reply:
There should be no issue with shared libs but once somebody tries to use `+crt-static` both `-lgcc_eh` and `-lunwind` would be passed. Depending on the order this could cause linking error or the resulting binary would have shared dependency.

Reply:
Oh right, I see

Reply:
Haven't tested with crt-static, but I think this problem should be ok now?

Reply:
Yeah, looks better.